src/Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier/Microsoft.VisualStudio.Debugger.Interop.UnixPortSupplier.cs

```diff
@@ -62,11 +62,24 @@ public interface IDebugUnixShellPort
         /// <returns>Home directory of the user.</returns>
         string GetUserHomeDirectory();
 
+        /// <summary>
+        /// Attaches gdbserver to a process.
```

Comment:
>Attaches [](start = 12, length = 8)

This feels a bit wrong to be in the port supplier to me. Is there a reason we have this here instead of the engine? (And expose out whatever port supplier functionality the engine might need to make it work).

Reply:
Oh I see, it is because liblinux has GDP server-specific stuff. Make sense.

---
In reply to: [165417036](https://github.com/Microsoft/MIEngine/pull/686#discussion_r165417036) [](ancestors = 165417036)